pkg/util/validation/limits.go

```diff
@@ -787,3 +790,59 @@ func MaxDurationPerTenant(tenantIDs []string, f func(string) time.Duration) time
 	}
 	return result
 }
+
+type userLimitsResponse struct {
+	// Write path limits
+	IngestionRate             float64 `json:"ingestion_rate"`
+	IngestionBurstSize        float64 `json:"ingestion_burst_size"`
+	MaxGlobalSeriesPerUser    float64 `json:"max_global_series_per_user"`
+	MaxGlobalSeriesPerMetric  float64 `json:"max_global_series_per_metric"`
+	MaxGlobalExemplarsPerUser float64 `json:"max_global_exemplars_per_user"`
```

Comment:
Many of these are `int`, not float. Why use float64 for everything? I'd suggest to keep the type same as original limit. (Same for read path and ruler limits)